tapir/shifts/emails/shift_understaffed_watch_mail.py

```diff
@@ -0,0 +1,60 @@
+from typing import List
+
+from django.utils.translation import gettext_lazy as _
+
+
+from tapir.coop.models import ShareOwner
+from tapir.core.mail_option import MailOption
+from tapir.core.tapir_email_builder_base import TapirEmailBuilderBase
+from tapir.shifts.models import Shift
+
+
+class ShiftUnderstaffedEmailBuilder(TapirEmailBuilderBase):
+    option = MailOption.OPTIONAL_DISABLED
```

Comment:
I think this mail should be required: if someone doesn't want to receive the mails, they would just unwatch.

The only case I can think of is someone going on vacation that wants to disable all the mails in one go without loosing track of which shifts are watched. Then I'd set the value to `OPTIONAL_DISABLED`, I think users would be confused by not receiving mails by default.

Reply:
agree